.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: lint
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
Could you briefly describe "synchronize?"

Reply:
Synchronize refers to when you commit changes to an existing PR.